distribution/docs/src/main/resources/_contents/_eventing/developing-custom-subscription-implementations-contents.adoc

```diff
@@ -0,0 +1,168 @@
+
+==== Developing Custom Subscription Implementations
+
+To create a subscription in ${branding} the developer needs to implement the `${ddf-branding}.catalog.event.Subscription` interface. 
+This interface extends `org.opengis.filter.Filter` in order to represent the subscription's filter criteria. 
+Furthermore, the `Subscription` interface contains a `DeliveryMethod` implementation.  
+
+When implementing `Subscription`, the developer will need to override the methods `accept` and `evaluate` from the `Filter`. 
+The `accept` method allows the visitor pattern to be applied to the `Subscription`. 
+A `FilterVisitor` can be passed into this method in order to process the `Subscription's Filter`.
+In ${branding}, this method is used to convert the `Subscription's Filter` into a predicate format that is understood by the Event Processor. 
+The second method inherited from `Filter` is `evaluate`. 
+This method is used to evaluate an object against the `Filter`'s criteria in order to determine if it matches the criteria. 
+
+[TIP]
+====
+The functionality of these overridden methods is typically delegated to the `Filter` implementation that the `Subscription` is using.
+====
+
+The developer must also define `getDeliveryMethod`. 
+This class is called when the an event occurs that matches the filter of the subscription.
+
+The other two methods required because `Subscription` implements `Federatable` are `isEnterprise` and `getSourceIds`, which indicate that the subscription should watch for events occurring on all sources in the enterprise or on specified sources. 
```

Comment:
This is a little confusing, I suggest `The other two methods required are `isEnterprise` and `getSourceIds`, because `Subscription` implements `Federatable`, which...`